### PR TITLE
Update App.cpp (fix function: create account)

### PR DESCRIPTION
**Pull request by DatDangg**

### CODE REVIEW

**Comment by tmquang0209** [comment]
what if the user unchecks the account type?

File: App.cpp [hunk 1]
```diff
@@ -1154,11 +1157,11 @@ void createAccount()
             break;
     }
     // if not choose type account => default is user
-    if (!isUser && !isAdmin)
-    {
-        type = "user";
-        isUser = true;
-    }
+    // if (!isUser && !isAdmin)
```
Comment (tmquang0209) [inline App.cpp:1160]: if the user unchecks the account type, this will set account type as user

**Comment by DatDangg** [comment]
oh, sorry, I forgot to open the comment there

**Comment by DatDangg** [comment]
I tried it and have forgotten =))))

**Pull request closed without merging**